modules/client-configuration/pages/kickstart.adoc

```diff
@@ -1,16 +1,18 @@
 [[client-cfg-reg-with-bootstrap-kickstart]]
 = Kickstart
 
+When you install a {rhel} client, there are a number of questions you need to answer.
+In order to automate installation, you can create a {kickstart} file with all the answers to those questions, so that no user intervention is required.
```

Comment:
```suggestion
To automate installation, you can create a {kickstart} file with all the answers to those questions, so that no user intervention is required.
```